app/checklist/views.py

```diff
@@ -88,3 +88,16 @@ def delete_default_checklist_item(id):
         flash('Error occurred. Please try again.', 'form-error')
         return redirect(url_for('checklist.index'))
     return redirect(url_for('checklist.index'))
+
+
+@checklist.route('/<int:id>/view', methods=['GET'])
+def view_all_checklist_items(id):
+    """View all checklist items, specific to each user"""
+    default_checklist_items = DefaultChecklistItem.query.all()
+    user_checklist_items = UserChecklistItem.query.filter_by(application_id=id)
```

Comment:
`application_id` is not necessarily the same as `user_id` since the `Application` and `User` models are different.

You might need to find the user with `user = User.query.get(user_id)` and then access the application with `UserChecklistItem.query.filter_by(application_id=user.application.id)` or something along those lines.

Reply:
Great!